ntoskrnl/inbv/inbv.c

```diff
@@ -161,7 +161,7 @@ BootLogoFadeIn(VOID)
     LPRGBQUAD Palette = (LPRGBQUAD)(PaletteBitmapBuffer + sizeof(BITMAPINFOHEADER));
     ULONG Iteration, Index, ClrUsed;
 
-    LARGE_INTEGER Delay; // Specified in nanoseconds
+    LARGE_INTEGER Delay = {{0}}; // Specified in hundreds of nanoseconds
```

Comment:
Does this needs to be initialized in all places, or it have only one field (QuadPart)?

Reply:
I'm sure it zeroes out the Delay because when the while() loop is run for the first time, KeDelayExecutionThread is called with a valid delay.

Reply:
Yes, specifying an initializer for any member of a struct, union, or array means the whole object will be initialized.
On an unrelated note, this comment seems completely unnecessary. The unit is obvious to anyone familiar with kernel programming, and anyone else needs to read the documentation for KeDelayExecutionThread anyway.

Reply:
Ok, removed.